Tests/ProjectDescriptionTests/TargetTests.swift

```diff
@@ -14,7 +14,13 @@ final class TargetTests: XCTestCase {
             bundleId: "bundle_id",
             deploymentTarget: .iOS(targetVersion: "13.1", devices: [.iphone, .ipad]),
             infoPlist: "info.plist",
-            sources: "sources/*",
+            sources: [
```

Comment:
this particular test intentionally had `sources: "sources/*"` as it was verifying `sources` is able to accept string literals (for backward compatibility).

For this particular PR, updating `test_toJSON_withFileList` should suffice